applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -8,13 +8,103 @@
+	BT_AUDIO_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT,                           \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_AUDIO_UNICAST_24_2_1
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_AUDIO_LC3_UNICAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT,                           \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#else
+BUILD_ASSERT(0, "Unsupported LC3 codec preset for unicast");
+#endif /* CONFIG_BT_AUDIO_UNICAST_RECOMMENDED */
+#endif /* CONFIG_AUDIO_SOURCE_USB */
+#endif /* CONFIG_TRANSPORT_CIS */
+
+#if CONFIG_TRANSPORT_BIS
+#define BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(_loc, _stream_context)                           \
+	BT_AUDIO_LC3_PRESET(                                                                       \
+		BT_CODEC_LC3_CONFIG(                                                               \
+			BT_CODEC_CONFIG_LC3_FREQ_48KHZ, BT_CODEC_CONFIG_LC3_DURATION_10, _loc,     \
+			LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 1, _stream_context),         \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 4u,     \
+					     20u, LE_AUDIO_PRES_DELAY_US))
+
+#if CONFIG_AUDIO_SOURCE_USB
+#if !CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED
+#error USB only supports 48kHz
+#endif /* !CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                    \
+						  BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_AUDIO_SOURCE_I2S
+#if CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                    \
+						  BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_AUDIO_BROADCAST_16_2_1
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT,                         \
+					     BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_AUDIO_BROADCAST_24_2_1
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT,                         \
+					     BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_AUDIO_BROADCAST_16_2_2
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_16_2_2(BT_AUDIO_LOCATION_FRONT_LEFT,                         \
+					     BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_AUDIO_BROADCAST_24_2_2
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_BROADCAST_PRESET_24_2_2(BT_AUDIO_LOCATION_FRONT_LEFT,                         \
```

Comment:
Why are all locations front left?

Reply:
A location has to be set, and front left is de facto standard. This can be overwritten in the application later